crates/bevy_ecs/src/query/iter.rs

```diff
@@ -545,7 +545,7 @@ impl<'w, 's, Q: WorldQuery, F: ReadOnlyWorldQuery> QueryIterationCursor<'w, 's,
             let index = self.current_index - 1;
```

Comment:
we should rename `QueryiteratorCursor`'s `current_index` field to `current_row` (ideally it could also be a Table/ArchetypeRow but i guess that's not possible :disappointed: )

Reply:
Done.